macros/edr/metadata_collection/get_columns_from_information_schema.sql

```diff
@@ -43,4 +43,17 @@
     from information_schema.columns
     where upper(table_schema) = upper('{{ schema_name }}')
 
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro athena__get_columns_from_information_schema(database_name, schema_name) %}
+    select
+        upper(table_catalog || '.' || table_schema || '.' || table_name) as full_table_name,
+        upper(table_catalog) as database_name,
+        upper(table_schema) as schema_name,
+        upper(table_name) as table_name,
+        upper(column_name) as column_name,
+        data_type
+    from information_schema.columns
```

Comment:
FYI: this query is going to be super slow - did you think about using a dbt-athena method to make that working? 
e.g. we can add another method in the adapter to leverage glue apis - those are going to be few order of magnitude faster

Reply:
Thanks for pointing this out. I noticed dbt-athena switched to glue apis some time ago. Definitely it should be checked, once the port is working properly overall.